### PR TITLE
Examples: used https, and added Codecademy Gamut

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## What I did

Sending a PR instead of filing an issue because I figured it'd be easier to discuss all in one place, and the last change to this examples list didn't have a backing issue either. Sorry if I'm bypassing any kind of system here!

`Airbnb Dates`, ~`Anyvision UI`~, and `Salesforce Lightning` were using `http` but could use `https` so I switched them over too. `Lonely Planet` is the last `http` link but its site doesn't seem to support `https`.

### CODE REVIEW

File: docs/get-started/examples.md [hunk 1]
```diff
@@ -34,25 +34,26 @@ NOTE for contributors: This is a curated list. Here's what qualifies:
...
-- [Salesforce Lightning](http://design-system-react-components.herokuapp.com/?path=/story/sldsaccordion--base)
-- [IBM Carbon](https://react.carbondesignsystem.com/?path=/story/accordion--accordion)
-- [Shopify Polaris](https://main--5d559397bae39100201eedc1.chromatic.com)
-- [Airbnb Dates](http://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default)
-- [Lonely Planet](http://lonelyplanet.github.io/backpack-ui/?path=/story/styles--design-tokens)
-- [Wix Style](https://www.wix-style-react.com/?path=/story/introduction-getting-started--getting-started)
+- [Airbnb Dates](https://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default)
+- [AnyVision UI](https://storybook.anyvision.co/)
+- [AppNexus Lucid](https://appnexus.github.io/lucid/?path=/docs/documentation-introduction--introduction)
+- [Audi](https://react.ui.audi/)
```
Comment (jonniebigodes) [inline docs/get-started/examples.md:40]: @JoshuaKGoldberg it seems that if you switch to HTTPS in the AnyVision UI design system it throws a 404:

This is what I got when I visited the page with HTTPS:
anyvision-https

And with HTTP:
anyvision-http


If you're ok with it, could you revert it back?

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@JoshuaKGoldberg thank you for the time and effort you put into the pull request. One minor thing and we're good to go on this one. Really appreciate you taking the time and organize the examples.

Also on a side note. I've checked your Storybook (CodeAcademy) and it looks amazing. Loved your UI!

Let me know once that item is addressed and I'll gladly merge it.

Stay safe

File: docs/get-started/examples.md (hunk 1 above)
Comment (JoshuaKGoldberg) [reply docs/get-started/examples.md:40]: Aha, darn, I could have sworn I checked them all!

**Comment by MichaelArestad** [comment]
Thanks for adding Codecademy Gamut. @domyen what do you think about alphabetizing the examples? Where they in a particular order before?

**Comment by jonniebigodes** [comment]
@MichaelArestad they weren't alphabetized up to this point. New examples were added to the bottom of the list with each new addition.

**Comment by domyen** [comment]
@jonniebigodes is correct. @JoshuaKGoldberg can you add Codecademy to the end of the previous list? Then we should be good to merge the PR!

**Comment by JoshuaKGoldberg** [comment]
Sorry @domyen I don't follow, what is the "previous list" here?

**Comment by domyen** [comment]
The previous list is the sort order we currently have on the site. I'd expect this PR to have one line of code added for Codecademy.
image

**Comment by JoshuaKGoldberg** [comment]
Oh, sure, I've un-sorted it now @domyen. I think you and @MichaelArestad aren't quite in agreement on whether this PR should sort them 😄 .

**Comment by MichaelArestad** [comment]
@JoshuaKGoldberg This looks good to me. Thanks for adding the `s` on two of those URLs!

**Review by MichaelArestad — APPROVED** [review]

**Comment by jonniebigodes** [comment]
@JoshuaKGoldberg thank you for factoring in the feedback and addressing it promptly. We appreciate it. I've checked and it's good on my end. I'm going to merge this pull request and include your example.

Looking forward to your next contribution.

Stay safe

**Pull request merged**